src/main/java/net/minecraftforge/srg2source/extract/SymbolReferenceWalker.java

```diff
@@ -555,23 +555,10 @@ private boolean process(ImportDeclaration node) {
         return false; //Do not walk children, ignore it all
     }
 
-    private boolean process(InstanceofExpression node) {
+    private boolean process(PatternInstanceofExpression node) {
         // visit children in normal left to right reading order
         acceptChild(node.getLeftOperand());
         acceptChild(node.getRightOperand());
-        SimpleName pattern = null;
-        try {
-            pattern = node.getPatternVariable();
-        } catch (UnsupportedOperationException e) {
-            // Nom: this is nasty, but the only other way to check is to use internal code. Which is also bad.
-        }
-
-        if (pattern != null) {
-            IVariableBinding bind = (IVariableBinding)pattern.resolveBinding();
-            trackLocalVariable(pattern, bind);
-            acceptChild(pattern);
-            System.currentTimeMillis();
-        }
 
```

Comment:
Since this just does what PatternInstanceofExpression accept does. We can delete this function entirely and return true in the bouncer.
The SingleVariableDeclartion of the right hand will be handled elseware.